src/main/kotlin/io/emeraldpay/dshackle/upstream/UpstreamValidator.kt

```diff
@@ -34,9 +34,7 @@ abstract class UpstreamValidator(
 
     abstract fun validateUpstreamSettings(): Mono<ValidateUpstreamSettingsResult>
 
-    fun validateUpstreamSettingsOnStartup(): ValidateUpstreamSettingsResult {
```

Comment:
You can do this fun with a modifier `open` and just override it in `EthereumUpstreamValidator`